### ISSUE TITLE
Some duplicates in the data?

### ISSUE THREAD

**Issue by StefanDumont** [acdh-oeaw/histogis#13]
It seems to me that 3-4 states are duplicated - with identical time periods. E.g. Baden, Bavaria and Belgique. Or am I missing something?

**Comment by sennierer** [comment]
I ran through the data and taking name, time-period and geometry into account there are no duplicates in the data. However, there are duplicates (e.g. the mentioned Baden) if you take only name and time-period into account. https://histogis.acdh.oeaw.ac.at/shapes/shape/detail/9837 and https://histogis.acdh.oeaw.ac.at/shapes/shape/detail/7987 were imported from different sources and have slightly differing shapes.
I guess we can either resolve the issues in the data (decide for one source and go with it) or include a switch in the APIs that - if turned on - automatically removes duplicates. The latter solution would allow us to keep competing data/information.

**Comment by StefanDumont** [comment]
Ah okay, in fact I only compared it by name and time (we will offer this to the user in correspSearch for selection in the future). I saw that they were different sources, but I didn't see that they were (probably minor) different forms. I would also vote for allowing different forms, which makes sense from a historical point of view. In our use case, we could also specify the source if necessary, so that this is distinguished in the selection. However, the user then knows nothing more about these differences, i.e. to what extent the boundaries differ. In this respect, a parameter that removes the "duplicates" in the API would be quite useful for our use case.
